Library/Homebrew/test/livecheck/livecheck_version_spec.rb

```diff
@@ -28,7 +28,7 @@
     expect(described_class.create(cask, Version.new("1.1.6")).versions).to eq ["1.1.6"]
     expect(described_class.create(cask, Version.new("2.19.0,1.8.0")).versions).to eq ["2.19.0", "1.8.0"]
     expect(described_class.create(cask, Version.new("1.0,100:1426778671")).versions)
-      .to eq ["1.0", "100", "1426778671"]
+      .to eq ["1.0", "100:1426778671"]
```

Comment:
We could probably just remove the test with the colon since it's no longer used in cask versions, right?

Reply:
So remove all three checks for `1.0,100:1426778671`? (formula, cask, and resource `expect`).

Reply:
Yes, I don't see why they should remain given that colons aren't used any longer. Let me know if I've missed something though.

Reply:
> We could probably just remove the test with the colon since it's no longer used in cask versions, right?

Do we actually have an audit for this yet?

Reply:
Doesn't seem like there's an audit for this yet. I'm happy to open a PR if we want to add this.

Reply:
From last comments in https://github.com/Homebrew/homebrew-cask/issues/95207, sounds like we are waiting on audit before we can close issue.

This PR may mark off last task though.